### PR TITLE
[OPPRO-173] Set Velox batch size

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

Use below spark conf to control Velox batch size of scan, agg and join.
> spark.sql.execution.arrow.maxRecordsPerBatch

Depends on https://github.com/oap-project/velox/pull/32.

## How was this patch tested?
locally tested

### CODE REVIEW

**Comment by zzcclp** [comment]
Please rebase and re-run ch-backend-test with latest version.

**Comment by rui-mo** [comment]
Hi @FelixYBW, do we still need this PR?

**Comment by FelixYBW** [comment]
Yes, we still need it

**Review by FelixYBW — APPROVED** [review]

File: tools/build_velox.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
     echo "VELOX_INSTALL_DIR=${VELOX_INSTALL_DIR}"
     mkdir -p $VELOX_SOURCE_DIR
     mkdir -p $VELOX_INSTALL_DIR
-    git clone https://github.com/oap-project/velox.git -b main $VELOX_SOURCE_DIR
+    git clone https://github.com/rui-mo/velox.git -b wip_config $VELOX_SOURCE_DIR
```
Comment (FelixYBW) [inline tools/build_velox.sh:52]: commit to oap-project/velox?

Comment (rui-mo) [reply tools/build_velox.sh:52]: Yes, there is a PR for that, and this change is just for testing and will be reverted before merge. The gluten PR and velox PR will be merged together.

**Comment by rui-mo** [comment]
Merging this PR. Close performance was observed with 32k batch size. The testing DWRF dataset still had 10k stride size.

**Pull request merged**